src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
```

Comment:
I'm generally not a fan of an empty string after a class name. But I see that now many developers do this. So you can do it but do it in the same way in all classes.